guides/getting-started/getting-started-openshift.adoc

```diff
@@ -56,14 +56,13 @@ Once the command above completes you should see a message similar to this:
 
 [source,bash,subs="attributes+"]
 ----
-service/secure-keycloak created
+service/keycloak created
```

Comment:
Same as https://github.com/keycloak/keycloak-quickstarts/pull/304#discussion_r822275265 , using insecure way is not recommended in modern time. Hope we can keep secure way while fixing other issue in question.